### PR TITLE
Fix case-sensitive username check

### PR DESCRIPTION
**Pull request by crobibero**
Fixes https://github.com/jellyfin/jellyfin-plugin-ldapauth/issues/34

Thought this was already in, sorry!

### CODE REVIEW

**Comment by dkanada** [comment]
We might want this to be a configuration option since it technically improves security.

**Comment by crobibero** [comment]
Good thought, I’ll do that tomorrow

**Review by BaronGreenback — APPROVED** [review]

File: LDAP-Auth/LDAPAuthenticationProviderPlugin.cs [hunk 1]
```diff
@@ -218,7 +221,7 @@ private LdapEntry LocateLdapUser(string username)
                     {
                         foreach (var name in toCheck.StringValueArray)
                         {
-                            if (username == name)
+                            if (string.Equals(username, name, usernameComparison))
```
Comment (oddstr13) [inline LDAP-Auth/LDAPAuthenticationProviderPlugin.cs:224]: CodeQL suggests using `.Where(...)`, thoughts on that?

**Review by oddstr13 — APPROVED** [review]

File: LDAP-Auth/LDAPAuthenticationProviderPlugin.cs (hunk 1 above)
Comment (crobibero) [reply LDAP-Auth/LDAPAuthenticationProviderPlugin.cs:224]: I’m surprised CodeQL suggests adding Linq where it isn’t already being used, but I definitely wouldn’t do it here

**Pull request merged**